### PR TITLE
create bmarwell 3rd party user feature post in staging

### PR DESCRIPTION
**Pull request by dmuelle**
#2336

### CODE REVIEW

**Comment by dmuelle** [comment]
Preview looks good on [draft](https://draft-openlibertyio.mybluemix.net/blog/)

Screen Shot 2022-08-26 at 11 43 28 AM

**Comment by dmuelle** [comment]
preview looks good on [draft](https://draft-openlibertyio.mybluemix.net/blog/)
Screen Shot 2022-08-30 at 1 19 22 PM

**Comment by dmuelle** [comment]
Note that the title of his post is "Guide: How to develop an IBM OpenLiberty Feature"

I've altered that slightly  for our blog roll: "How to develop an Open Liberty user feature"

Before proceeding, just want to make sure that's ok and that we are ok with syndicating something that says "IBM OpenLiberty"

File: blog_tags.json [hunk 1]
```diff
@@ -297,7 +298,7 @@
         },
         {
             "name": "community",
-            "posts": ["open-liberty-semeru-performance",
+            "posts": ["develop-user-feature.adoc","open-liberty-semeru-performance",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["develop-user-feature","open-liberty-semeru-performance",
```

File: blog_tags.json [hunk 2]
```diff
@@ -325,7 +326,7 @@
         },
         {
             "name": "developer experience",
-            "posts": ["open-liberty-with-cargo-tracker",
+            "posts": ["develop-user-feature.adoc","open-liberty-with-cargo-tracker",
```
Comment (mbroz2) [inline blog_tags.json]:
```suggestion
            "posts": ["develop-user-feature","open-liberty-with-cargo-tracker",
```

**Review by mbroz2 — CHANGES_REQUESTED** [review]

**Review by mbroz2 — DISMISSED** [review]
Approved; however, I'll mention that Maven has "user-feature-install" twice (and my personal preference is for spaces after commas)

**Comment by dmuelle** [comment]
Thanks @mbroz2 - added spaces and removed the redundant tag from the Maven section

**Review by ramkumar-k-9286 — APPROVED** [review]

**Pull request merged**